File: public/usage-examples/json/json_from_color/json_from_color-1-find-purple.py
from splashkit import *

# Create a color
clr = color_purple()

# Convert the color to a JSON object
json_obj = json_from_color(clr)

# Display the JSON object as a string
write_line("JSON from Color: " + json_to_string(json_obj))

# Free the JSON object
free_json(json_obj)

File: public/usage-examples/json/json_from_color/json_from_color-1-find-purple.cpp
#include "splashkit.h"

int main()
{
    // Create a color
    color clr = COLOR_PURPLE;

    // Convert the color to a JSON object
    json json_obj = json_from_color(clr);

    // Display the JSON object as a string
    write_line("JSON from Color: " + json_to_string(json_obj));

    // Free the JSON object
    free_json(json_obj);

    return 0;
}
